packages/shared-components/QueueItems/Header/index.jsx

```diff
@@ -65,6 +74,17 @@ const Header = ({
           {renderCalendarButtons && (
             <CalendarButtons onCalendarClick={onCalendarClick} />
           )}
+
+          {renderCreatePostButton && (
+            <CreatePostButtonWrapper>
+              <Button
+                type={'primary'}
```

Comment:
A very tiny one, but the curly braces aren't necessary for this line:
```suggestion
                type="primary"
```